src/langs/en.json

```
@@ -248,6 +248,12 @@
   "STATUS_TOOLTIP_IS_COLLAPSIBLE": "Collapsible",
   "STATUS_TOOLTIP_SHOW_CHATBOX": "Chatbox visible",
   "SETTINGS_FILE_SETTINGS_TITLE": "File Settings",
+  "DELETE_MEMBERSHIP": "Delete Membership",
+  "DELETE_MEMBERSHIP_MODAL_CONFIRM_BUTTON": "Confirm",
+  "DELETE_OWN_MEMBERSHIP_MESSAGE": "Are you sure you want to delete your own permission on this item?",
+  "DELETE_MEMBERSHIP_MESSAGE": "Are you sure you want to delete membership on this item?",
```

It would be nice to include a little more information in the message. I think this information you already have in the `membershipToDelete` object. 
The values can be passed to the translation function like this: 

```ts
dialogText = translateBuilder(BUILDER.DELETE_MEMBERSHIP_MESSAGE, { name: membershipToDelete.member.name, permissionLevel: membershipToDelete.permission });
```

```suggestion
  "DELETE_MEMBERSHIP_MESSAGE": "Are you sure you want to remove {{permissionLevel}} access on this item for {{name}}? They will loose access to this item.",
```